File: camera/src/main/java/com/ys/camera/listener/CaptureListener.java
package com.ys.camera.listener;

/**
 * author : liutiantian
 * e-mail : Tareafengye@163.com
 * version: 1.0
 */

public interface CaptureListener {
    void takePictures();

    void recordShort(long time);

    void recordStart();

    void recordEnd(long time);

    void recordZoom(float zoom);

    void recordError();
}

File: camera/src/main/java/com/ys/camera/CaptureListener.kt
package com.hbzhou.open.flowcamera


/**
 * author : liutiantian
 * e-mail : Tareafengye@163.com
 * version: 1.0
 */
interface CaptureListener {

    fun takePictures()

    fun recordShort(time: Long)

    fun recordStart()

    fun recordEnd(time: Long)

    fun recordZoom(zoom: Float)

    fun recordError()
}